tests/functional/db/cram/import-query-export.t

```diff
@@ -0,0 +1,58 @@
+Setup
+
+  $ export AUGUR="$TESTDIR/../../../../bin/augur"
+
+Import metadata and sequences.
+
+  $ cat >in.tsv <<~~
+  > strain	date
+  > SEQ1	2000-01-01
+  > SEQ2	2000-02-01
+  > SEQ3	2000-03-01
+  > SEQ4	2000-04-01
+  > ~~
+
+  $ cat >in.fasta <<~~
+  > > SEQ1
+  > ATAT
+  > > SEQ2
+  > CGGC
+  > > SEQ3
+  > ACGT
+  > ~~
+
+  $ ${AUGUR} db import \
+  >  --metadata in.tsv \
+  >  --sequences in.fasta \
+  >  --output-db data.sqlite3
```

Comment:
Hmm, I'm having second thoughts here. I think these subcommands are trying to do too much at once. Having `--metadata` and `--sequences` at the same time makes sense for other subcommands where both are needed, but that's not the case here.

What about a modified CLI where each subcommand acts on file types rather than Augur-specific "metadata"/"sequences", and only one at a time?

```sh
augur db import csv \
    --file metadata.tsv \
    --table metadata \
    --output-db data.sqlite3

augur db import fasta \
    --file sequences.fasta \
    --table sequences \
    --output-db data.sqlite3

augur db export csv \
    --db data.sqlite3 \
    --table metadata \
    --output-file metadata.tsv

augur db export fasta \
    --db data.sqlite3 \
    --table sequences \
    --output-file sequences.fasta
```

This way, the path to support NDJSON and chaining with `augur curate` as mentioned in https://github.com/nextstrain/augur/pull/1094#discussion_r1050282099 is much more clear.

```sh
augur curate passthru \
    --metadata metadata.tsv \
| augur db import ndjson \
    --file - \
    --table metadata_and_sequences \
    --output-db data.sqlite3

augur db export ndjson \
    --db data.sqlite3 \
    --table metadata_and_sequences \
    --output-file - \
| augur curate normalize-strings > records.ndjson
```

Anyways, this might be premature thinking since we haven't even defined a use case for the `augur db` CLI. I should be focusing more on integrating the underlying class interfaces into `augur filter` which is a well-defined use case...

Reply:
I'm leary of `--table`. I don't think that sort of general interface is what we want here. I think we want a specific "Augur database" format, the same way we have a specific "Augur node data" format or "Nextstrain dataset" format.

> Anyways, this might be premature thinking since we haven't even defined a use case for the `augur db` CLI. I should be focusing more on integrating the underlying class interfaces into `augur filter` which is a well-defined use case...

Maybe let's get a small group of interested folks to discuss directions/use case/intentions here?

Reply:
> Maybe let's get a small group of interested folks to discuss directions/use case/intentions here?

Sure! I want to put the newer work with `augur filter` out for review first (as mentioned in https://github.com/nextstrain/augur/pull/854#issuecomment-1487559267) - with that it should be easier to have this discussion.